control/control.yaml

```diff
@@ -165,4 +165,39 @@ components:
           description: >-
             Protocol specific states
           type: string
-          enum: [start, stop]
\ No newline at end of file
+          enum: [start, stop]
+
+    Device.State:
+      description: >-
+        Sets attributes for the requested state/actions to be performed on device(s)
+      type: object
+      properties:
+        choice:
+          type: string
+          enum:
+          - lacp_pdu_transmit_state
+        lacp_pdu_transmit_state:
+          $ref: '#/components/schemas/LacpPduTransmit.State'
+
+    LacpPduTransmit.State:
+      description: >-
+        Set LACP PDU transmission state for LAG Member Port(s).
+      type: object
+      required: [state]
+      properties:
+        port_names:
+          description: >-
+            The names of LAG member port objects to control.
+            An empty or null list will control all LAG member port objects.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - '/components/schemas/Port/properties/name'
```

Comment:
this should be /components/schemas/Lag.Port/properties/name

Reply:
Done